app/fetchers.ts

```diff
@@ -0,0 +1,39 @@
+import { ViewService, jsonOptions } from '@penumbra-zone/protobuf';
+import { createPromiseClient } from '@connectrpc/connect';
+import { createChannelTransport } from '@penumbra-zone/transport-dom/create';
+import { assertProvider } from '@penumbra-zone/client/create';
+import { bech32mAddress } from '@penumbra-zone/bech32m/penumbra';
+import { getMetadataFromBalancesResponseOptional, getAmount } from '@penumbra-zone/getters/balances-response';
+
+const createFetchClient = (wallet: string) => {
+  const provider = assertProvider(wallet);
```

Comment:
Can you handle the "not installed" case? That is, if a user does not have Prax, they are given a button to direct them to the chrome web store to download it.